### PR TITLE
Add Node version information for 4.x.x

### PR DESCRIPTION
**Pull request by jasonbellamy**

### CODE REVIEW

File: user/languages/javascript-with-nodejs.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ You can choose Node.js and io.js versions to run your tests by adding the follow
 These values are passed on to [`nvm`](https://github.com/creationix/nvm);
 newer releases not shown above may be used if `nvm` recognizes them.
 
-This will make Travis CI run your tests against the latest version 0.6.x, 0.8.x, 0.10.x, 0.11.x, and 0.12.x branch releases,
+This will make Travis CI run your tests against the latest version 0.6.x, 0.8.x, 0.10.x, 0.11.x, 0.12.x, 4.0, and 4.1.x branch releases,
```
Comment (BanzaiMan) [inline user/languages/javascript-with-nodejs.md]: Let's use 4.0.x for consistency.

File: user/languages/javascript-with-nodejs.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ You can choose Node.js and io.js versions to run your tests by adding the follow
...
 as well as the io.js 1.0.4 and the latest io.js stable release.
```
Comment (BanzaiMan) [inline user/languages/javascript-with-nodejs.md]: io.js versions should be modified, too.

Comment (jasonbellamy) [inline user/languages/javascript-with-nodejs.md]: How should we handle the io.js reference now that's its combined with Node.js? I believe the last version published under io.js was 3.3.1. Something like:

```text
...as well as the latest io.js stable release.
```

?

Comment (BanzaiMan) [inline user/languages/javascript-with-nodejs.md]: That sounds good to me. Thank you.

**Pull request merged**